src/ShuttleAutomation.cpp

```diff
@@ -181,3 +181,72 @@ void ShuttleSetAutomation::DefineEnum( int &var, const wxChar * key, const int v
    if( bWrite && bOK)
       var = temp;
 }
+
```

Comment:
In the previous commit, this renaming happened:  ShuttleParams ---> SettingsVisitor

This class is called ShuttleDefaults and used to inherit from ShuttleParams - now called SettingsVisitor.
By analogy, this class should now be called SettingsDefaults, or SettingsVisitorDefaults maybe?

Reply:
I could rename things too, but I thought that might make the PR more overwhelming.  But if you don't mind it, I will add a renaming commit.

Reply:
I think subclasses of abstract visitors should be named for what they do.  This one should be called "SettingsLoader".  I would rename

ShuttleSetAutomation -> SettingsSaver
ShuttleGetDefinition -> SettingsDescriber
ShuttleDefaults -> SettingsResetter

Reply:
Fine for doing that on another PR